R/pkg/R/column.R

```diff
@@ -132,17 +132,24 @@ createMethods()
 
 #' alias
 #'
-#' Set a new name for a column
+#' Set a new name for an object. Equivalent to SQL "AS" keyword.
```

Comment:
Moving to `generics.R` sounds good.  "Column or SparkDataFrame" in place of "object" as well.

Regarding "AS"... In SQL it can be used with both expressions and tables so I deliberately didn't quantify this with `Column`.

I am not sure if we really need to state that it returns a new object. Maybe  _Return a new Column or SparkDataFrame with an alias. Equivalent to SQL "AS" keyword._? But it doesn't sound great.